mesonbuild/coredata.py

```diff
@@ -323,6 +330,7 @@ def get_builtin_option_default(optname):
     'layout':          [UserComboOption, 'Build directory layout.', ['mirror', 'flat'], 'mirror'],
     'default_library': [UserComboOption, 'Default library type.', ['shared', 'static'], 'shared'],
     'backend':         [UserComboOption, 'Backend to use.', backendlist, 'ninja'],
+    'components_default_value':  [ComponentOption, 'Value to use by default for component options.', 'auto'],
```

Comment:
What I was trying to say was that to me the description sounds like it will force this value on *all* components, not just the autodetect ones :)